src/pages/elements/pictograms/design.mdx

```diff
@@ -35,6 +35,17 @@ and work well in presentations and marketing communications.
 
   </ResourceCard>
 </Column>
+<Column colMd={4} colLg={4} noGutterSm>
+    <ResourceCard
+      subTitle="Pictograms Master"
```

Comment:
Do we want "Master" to be capitalized?

Reply:
@abbeyhrt it should be lower case (Sadek and Mike agree)

Reply:
changed to lower case